From asterisk-gui-$(version)/README:

 Use of Graphics, Trademarks and Logos
 =====================================
 The images and logos included with Asterisk-GUI may only be distributed with
 unmodified versions of the toolkit and may not be used in any derivative works
 without special written permission from Digium.

This is the reason we replace the images.

--- debian/images/digiumlogo.gif.uu
+++ debian/images/digiumlogo.gif.uu
@@ -0,0 +1,57 @@
+begin 664 digiumlogo.gif
+M1TE&.#EAC``N`/<```````H*"A,3$QL;&R,C(RHJ*C(R,CL[.P`O=``W>0`[
+M>P!`?41$1$I*2E-34UM;6V-C8VMK:W)R<GM[>P!%@@!(A`!.B`!0A0!2BP!8
+MC@E8C0!6D`!;DAQ>D0!AE0!EF`UCE0YGF1!@DAUJER1LFB-RGBAUGC%QGR1T
+MH")YI2MTHC)\H3EVHC6!ISF`IDZ'JD^0L4R8NE.-L5R4LV*4M&.9M6><N&V>
+MO&>AOVVBN7FCOG&CP'FBP'^KP7>VS[0`!;P$$;\`&\$`!,,`#<0`%,4`&\8`
+M(L4"*\4-*<@`),H`*\H,+,().,L`,L03(L82,LH:.M(?.<HF*,8K.\@E.\P<
+M0LDB0,LK0\TK2,TP0\XQ2L\^3\\T7,X\5-$U7-`[7=@Y5=%$7,M+9=1-8--+
+M:]Y!8=%59-%:>,ED<M9A?MQB@MYLAMILB=5TCM5[B]EPA-QWBMU[D>%]F(.#
+M@XN+BY*2DIR<G*.CHZJJJK.SL[N[NX2LP)"SQ9&TRI6ZSIJUR)NZS*W,V;3-
+MV[[-UKS2W*70X->$FMF+GMF0I=F7JN"$G."*G^J"E>.4I>"GMN2LO.JHNNBR
+MOO6BLOFQNN.PP.BTQ,/#P\O+R\K#T=_$SM/3T]O2V=W=W</7X<3;Y</>ZLK;
+MX\+E[<?N]M3DZ]SG[=WI[][L\>C#S^O$T._*U.W3VO+)S_''T_+2U/'1V?+:
+MWO#:XN+BXN7M[NKJZN;O\>WS]N_W^//DZ?3I[?CH[?3P[O/S\_'V^/3Y^_KR
+M]OCU^_CX]_[^_@``````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M`````````````````````"'Y!```````+`````",`"X```C^`(,)'$BPH,&#
+M"!,J7,BPH<.'$"-*-`AJUL2+&#-JW(BPUAY>'$.*'$DR6*\3H$JJ7,ER(8T:
+MMEK*G*D24(>4-'.2!.;JEL^?L#)VTG#CE\ZC(7&)Z1*F:=,N:2Z.(@'"$]*K
+M&QL5*9*D:]<B;232:E%A!]:S&0T5:<*6K1(BE2#R@F$!A56T>"6^(=+6;1%9
+M#WM@N&`VK^&'O,SP[:O$B4-"%CBH&'6X<D-55Y+T;9*D"D-<"CYDL&&Y],)'
+M3Y1L+L)EX8(/'%CL4FCI#ITZ>2Q:UF.G#NZK<I*H[ILD3D(+'F#G0)@'@//G
+MSR'H'LBI`/3KSQEH0FA'`/;G`V;^"]PUY[OS.0;S!("^$,_UA6F,;$Z"Q-%!
+M$QE@GS@(;,#S``($N-YSEA`T"P/.!:#@@@I"AQY!NUS'8(,,B*<)=``*"!TP
+M!%EBW7.7*'1`@@`8P-`8:Q%WQ2L%W8#!!QYPL(=!$2;(`!Z<[!*+'@\,",!V
+M`AWHG`-XV&&DD71$\*$`>0Q4(P`"'#!!'4<:^5LPG#PGP`-ZS*+C'0?X*%XP
+M'@(P(`$)[4(`=`4P],L5\O551!>X#`3(!1_`1L)!")IIQT%Z^&?F0$("4`="
+MLPPHP4`CFEG@0@,6\&A!=#S7P$!E$B!H0A,XUV>;#)VRQ'!M)4%&+L$0LD&>
+ML`%BD";^@O[)W8`A!A-+GW0@!$RG`#Q@D1X#-KF0)0G6>I`$S@W`B4"76#>`
+M'>L]@-"(`;@'`*@,07($J4TH$80DH83`09X<[&D0!,\MM"8`!P2):T)U.-=`
+M+,$XD"Y#'S*PT',3"%1F`+L,>%`>_M%QAW/8,K2MG&5@DL*X>59`")_.2:M0
+M`_?>ZERN_#W@'`0"&>!<!`T]QW%"W@%P*9D?[F(O`+(2%(%SL=B!L$.GI,A6
+M$F!L$@,'R>EYT"XB&[I0><ZYZ]RA!\6[=#":?#AI0K,D./5!+[?+,LU5`X"F
+M@0@.$$R\`22LD!6D)A'%*CX`S:H"@+WZH1X+V9QT,(4:`,'^`WSSS0`#*3LP
+M6Z#.39<0L68NJ]#,[/K[(;WK&COV>G>,?3-#E.BLQ!*3%.(VN28@U*QSDA^$
+MA\!X]VE>LG/HED?*8QZ>H.''.J?UO\L>7")!Z/YH^;4-47%$J8Q\\GF>%B14
+MG7-T*W0'ZH7^)V&_`Q%<.$.(!Z!X0L@VOC4`BCLG@&YE^OJ[V0<A$F<317PA
+M"@<0LXH!'X@6'3-"2`,@D,8`<`R,4<#H'@`JQ*S(,206"6I>0C`V0,?13"`,
+M)%DP*@4`83D-?0:9PG",P`2!`(("K'I;W`S2)Y`IY&7Z2]W&^!.=@11-@OMZ
+MFD($I:_OT4L@SC'1+.PU@%I=<"'^;!B.$HQ`$![@*80?4`!"/':WA*S+8H4Z
+M64&L98!:J:XA`Q*;0A+$L3(!X(;!6$\`[@`L`#SH=R9*B"^FH)FWQ&0@M'A!
+M?D+(@0L`*F65FY5S%!=%A"#03'@0R.EDZ#SQE8X@%!P`^1XW$+LUP&Z!%,@/
+M$[(&U;PE$P;Q!`KB1ZX44(QF'"((,#2Q+BW:ZET'B<6'3L9``%R"=@8!#QB=
+MI(?GP-"+LY3>O`8RR8/`0@M*4$(1#(&00(@@:*SBP`P,,CKG2,`2>K`$-`5H
+M2A7V#U$,/./R/%4'/.`A#WGX)AX>92TST2&:T.21I3K$R('TCI#GHZ01A!F5
+MA/@!F>3^VD`?#&()U7TG``>8Y2P:=<:"[*)WU/-7*\U3`/'4X4/F"8`),;6N
+M[07C$N`1ED`HB,%@I`).12`#0W20`4YR0`2",,@L)N```OBH;`]@FH$>H*G[
+MB;([`RBH0.80`088H`!`#>H`'A!*J$&``0-XZ0$B8-.+-H``!1#H`30E@=C9
+M80`%6)E!X&"$]L5.5S+@)&Q*0!D:6:(WM[G#(04R2FG.LB"SB*8FBDJ06&CB
+M$M*4)EZ!5-<\H-4.>."K0?%JB:_&0IH6Q5LTUXJ**A#A"JIP2"U&(-8,N(`4
+MILGL0.(P!"A$`B*DR``^13.#6FC6-*PX0A$6(1%3A(:.&:!8WVDK`PPW"$$-
+M%QD$`BK`VPI8@`()^,-L#^.*'XPA([Q(KG*32]?A8N47:)!"+YQ+W8BT`@A&
+DJ:YV&>(++R1BN^!5B"O.$-[R'N00YDTO0::KWO:ZU[P!`0`[
+`
+end
--- debian/images/refresh.png.uu
+++ debian/images/refresh.png.uu
@@ -0,0 +1,24 @@
+begin 664 refresh.png
+MB5!.1PT*&@H````-24A$4@```!`````0"`8````?\_]A````!'-"250("`@(
+M?`ADB````!ET15AT4V]F='=A<F4`=W=W+FEN:W-C87!E+F]R9YON/!H```,B
+M241!5#B-;5)-;%1E%#WW>^^5^:TS;0=;J+8*AA*FH:(SC7_1F$8WQ&!8J+Q$
+MC)"!\+-A!QL,30P1=&&BX`.C,4PEF#0!-_P$(9+4PJMQT="2U-+&`48[M!WH
+MS+3SWON^ZZ+SDEEPDIO<Q3TG)^=<8F;X2&6RSS3HVA$0WO2D:M6$F"?"N./*
+M$P`NV);IUMT>!'""?(%T)KM%T\1`3]<J?5UGL]'6%`&842A6,#)VOWQG^J$G
+M%>^V+?-L>E<VPXSO`,2(F9'*9-<;NAC9VM<=BL>"T(G0'`T@'-`1#1D(&@(S
+MQ44<SPY7"L6*'0TU]#XN5PUF-!$SX]4]9T_UK&O[-!@PQ*W1'*JN!T/7Y)K5
+ML<6^]//AWO6MI`F"$(1?KDVHMUYL%P>^ONHXKFS1`8"9/[PS71"NIU@Q+S+C
+M,\>59\:G9Y,3]^:/7N]H[MJW=5,H$0OAXW>[1"T&`J`$`$BIPE7'\Y(O//N#
+ME.H1@$';,O.V95[YX]N/7AJ=+.S=?>P2[C\L87JF#*D8``,`"RROJNFI2']C
+M)/!59_O*;@!3=6GKNB9V;GLGZ78^'46B<04T0;X#UFMW\5\_W[R`)^.T)]5K
+M%X<GJU?MJ27?O..J``#V6Z@",&H$KIO?`.P$X#Y!6+<M\YXOX)T_^KX&$)@9
+M%^U_U*GS?^6D5#VV91;K68>S8ZM*Y:6.WT?&;P"(Z0!`!!`1[OY7AB$8)P?_
+M)`!?U)-3F6R8B.[V;EQ[)C\SEP*@`:@*`!!$!``=B1!6MX3QY?ZWJ:TE<NR5
+M/3_?2F6RZ9K&]A6&KN?RLP?RA>+KFD:N;9FN6'9`>%QQ<6%HBCV/T;XRBL,[
+MW@A]T)=\N;4Y<BV]:\`U=/&-5#(^]V@!38U!TH08!`#_D:C_QZ&EB=S<Y*7A
+MR<[,EDWA1"R([K4)>JX]'BJ67?P[6X+C>!0)&CAW>;3B274$`)8?23%-Y.9.
+MWSRY+?F@L/!)__<WYH\/W"P-W\XC/UM&U9401"C,E^2YRZ.+4JF,;9EC`."W
+M\!.`[;9E<BTP`\![#8:V5S%OD%+%#5V;(<)0U9&';,O\VP_W?RY5=FH39<S&
+,`````$E%3D2N0F""
+`
+end
--- debian/images/favicon.ico.uu
+++ debian/images/favicon.ico.uu
@@ -0,0 +1,84 @@
+begin 664 favicon.ico
+M```!``(`$!````````!H!0``)@```"`@````````J`@``(X%```H````$```
+M`"`````!``@``````$`!`````````````````````````````/___P!3*N@`
+MO8<%`*^/H@#DT*T`B&K?`,>?5@"YJ.T`(@#&`-W5]0">6@``U[N'`*&)Y@#S
+M[-X`0Q/.`+B&*@!F/]8`[NOZ`*YU#P"_DT$`MZ+6`%@NTP#1L74`UKZ<`#0%
+MT0!\6]P`J&L``/KV\`"LE^D`3B#1`)=^XP"R=P$`LWT:`/CT_P!`$=H`RZ5?
+M`"T`R`#DW?@`CW+B`-;-]`"SH.P`PIA)`&`XU0#\^O@`I&,``-K`C0!'&L\`
+MQ9Q/`#\-S0`W`\L`O9`[`/CRZP#2LWL`FX+D`#(!R@"J;@0`__K_`(INX`"F
+M9P``__WW`/'N^P#[^?0`]?3\`/7NX0"N=0L`2Q[0`+>E[`#X]_T`T[1X`/CT
+M[0`P`,D`IVD!`/W[^@"-<>$`I64``/?U_0#___T`J&@``/[^_@`S`,H`\.W[
+M`/3LWP##F$@`N*;L`"P`R``R`,H`IF8``*=I``#___X`^//K`._K^@#DT*P`
+MT;%V````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M```````````````````````````````Y`0$!`0$!`0$!`0$!`0$`(A]$`0$!
+M`0$!`0$T6D`!``%4&@$!`0$!`4\P$S@760`!`1$&`0$!`0$D2RTJ20$``0$*
+M)0T!`0$7.SLS+`$!``$!`2<)*4T,.QL01@$!`0`!`0%1+QD5`UA!4@$!`0$`
+M`0$!`4,W(P0@2%P!`0$!``$!`0$H,58"&$XA/@$!`0`!`0$F'C(R"`$N"T4!
+M`0$``0%;%E!0'0$!`5U7#@$!``$]*U5'-@$!`0$!!Q0!`0`!.@]"2@$!`0$!
+M`0%3!0$``1),/P$!`0$!`0$!'#4\``$!`0$!`0$!`0$!`0%960``````````
+M``````````"`````@````(````"`````@````(````"`````@````(````"`
+M````@````(````"`````@````(````#__P``*````"````!``````0`(````
+M``"`!`````````````````````````````#___\`73?D`,*1)@"PGNH`V+N)
+M`"``Q0"?7```XM3,`(1FW@#)HET`SL/R`$88S@#FX?@`KW85`)J"Y`#CSJL`
+M\^K;`&]+V0#`E$(`.`/,`(IR\P#PX[\`T+!V`%8KT@"K;P``O:_M`-[3X0#X
+M\^T`W<2;`,N?-P"WA"X`VM/W`&A&Z0"ED.<`O(<:`'E8VP#T\OP`C7+@`.;4
+MN`#LW\L`O(TX`+-^'@#-J6P`Q9Q3`#\.S0#'NO``*P#(`-.T@`"E90``32#0
+M`.WI^@"WINL`^_CW`.?7P@#DV=L`42C:`*US#`#4RO,`:D37`+-[#P#U[N(`
+MS*=D`'11V@!_8-T`E7SC`)^(Y0#QYM4`WLBA`/_^]P#Z^OX`MH,G`,*T[@`R
+M`,D`/`C-`*F5YP!$$LX`U+5Z`*MP!@#.K'$`J&H``*YT``!3)M(`WM?V`+)\
+M&0#Z]O$`ZMO$`'Q<W``X",H`NJKK`-O"EP"CC.4`]O#F`+F(+@!'&=(`^?;_
+M`%<OTP#1L7H`_?SZ`,>>5@#W]?P`4"31`.[BS`#9T?0`G87F`#4`RP#6S?,`
+MT,;R`.G9P`"J;@,`0@_.`$45SP"T?QL`WL><`"@`QP`^"\T`^O?T`.S>R`#'
+MGU,`2!K/`-S5]0#=QI\`.@7,`*%>``"C8P``IF@``*QR``#]_?\`K7(&`+6D
+MZP"RH.H`_/KY`*YT#@"Q>A@`]_+K`/+IW0"]CS<`Z->_`,.U\`#2L74`-@++
+M`#@%RP`Z",P`J6P``&U,V0"]K>P`SJML`,ZK;P#7NXL`,`#)`$(1S@`Y!\L`
+M_O[^`/S\_@#[^_T`NXPW`#$`R@`S`,H`-0'*`#D$RP`[!LP`I&0``#P'S`"F
+M9@``IVD``#T)S0"J;0``_?OY`*MQ```_#,T`K7,``$`.S@#Z]_(`^O7P`.SI
+M^0#`DT$`S</Q`'-1VP#!L^X`>%C<`-['H`"#9=X`L9_J`(QQX``I`,<`,0#)
+M`#(`R@`S`,D`-`#+`#0`R@#___X`-0#*`/[__@`W`LL`I68``*9G``"G:```
+MJ&D``#D%S`"H:P``JFX``*IO``"K<```]//\`#L(S`"L<0``K70``#T*S0`^
+M"LT`00[.`$$/S@!"$,X`0Q'.`$,2S@#QY]4`W-3U`,_&\@"\C3D`YM2W`,RG
+M90"PG^H`I9'G````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M`````````````````5\!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0`!
+M&M@!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!``$ELW@!`0$!`0$!`0$!
+M`0$!`0$!`0$!`0$!`0$!`0$``0&*93,!`0$!`0$!`0$!`0$!`0$!`0$!`0$!
+M`0$!`0`!`0&Q8&0!`0$!`0$!`0$!`0$!`0%#84U-34V3=0$!``$!`:Y8D)H!
+M`0$!`0$!`0$!`0$!/2I\?7U]!RD<`0$``0$!`5L&0+X!`0$!`0$!`0$!`:U=
+MI,ZJSL39-0$!`0`!`0$!9!BXMP$!`0$!`0$!`0&GK\.JJLY01X8!`0$!``$!
+M`0$!64F5#P$!`0$!`0$!`6/"JJJJCX6'`0$!`0$``0$!`0'`.[R]2P$!`0$!
+M`0$KPZJJJL@Y*`$!`0$!`0`!`0$!`0$ZCDJ>@0$!`0$!,*-^JJH9;2<!`0$!
+M`0$!``$!`0$!`0&UG="72`$!`5K$S:JJJ,5$`0$!`0$!`0$``0$!`0$!`0UO
+M<\PM"[YYC\JJJJJDE`$!`0$!`0$!`0`!`0$!`0$!`6B<U,9>(!ZA4:JJ4$\!
+M`0$!`0$!`0$!``$!`0$!`0$!RU)ZU,$X&P.DSJI0/@$!`0$!`0$!`0$``0$!
+M`0$!`0$!D9[2U+\"-R/'SGZ/5@$!`0$!`0$!`0`!`0$!`0$!`0$!),'3U+LA
+M"#RFSL>(F`$!`0$!`0$!``$!`0$!`0$!`482C=/33'(5%D[)JL-Q`0$!`0$!
+M`0$``0$!`0$!`0$!5YW4T]04,F<!$,?*R'"L`0$!`0$!`0`!`0$!`0$!`2:Y
+MT]/4H-5J`0$!'5#-PXL!`0$!`0$!``$!`0$!`0%"G-'3EJ7,+@$!`0$!!</)
+M@!$!`0$!`0$``0$!`0$!!+VIT].IC#0!`0$!`0$!%\/#=@$!`0$!`0`!`0$!
+M`;*-SY;3J[HB`0$!`0$!`0$!VS%0-@$!`0$!``$!`0%K+:+4TVZY00$!`0$!
+M`0$!`0$!+'N;?P$!`0$``0$!4W>?EM/3E0D!`0$!`0$!`0$!`0&#$P>T`0$!
+M`0`!`=<,N&F]O2\_F0$!`0$!`0$!`0$!`0%5'X1T`0$!``$!L-V"MK8$W&0!
+M`0$!`0$!`0$!`0$!`0%<#I(!`0$``0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!
+M`0'65&8!`0`!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0%L"F(!``$!`0$!
+M`0$!`0$!`0$!`0$!`0$!`0$!`0$!`0':B0$``0$!`0$!`0$!`0$!`0$!`0$!
+M`0$!`0$!`0$!`0%%`0``````````````````````````````````````````
+M@````(````"`````@````(````"`````@````(````"`````@````(````"`
+M````@````(````"`````@````(````"`````@````(````"`````@````(``
+F``"`````@````(````"`````@````(````"`````@````/____\`
+`
+end
--- debian/images/tick.gif.uu
+++ debian/images/tick.gif.uu
@@ -0,0 +1,17 @@
+begin 664 tick.gif
+MB5!.1PT*&@H````-24A$4@```!X````>"`8````[,*ZB````!F)+1T0`^0#Y
+M`/G"$XNT```"*4E$051X7K76W6M6!1S`\<]YWMI3*"&#%;$K$;SQ)@J[[DH"
+M'4H1"4/I;1AMSPB5X28<-U-W43QM<V$WW4117D06"4&CJR!8U)6"5VJ@X-LD
+MMLY\YCE'\"H.!]GS[#R??^#[^_VN?D%O;Z^N^]!V%=NERA*7S;I4T4T-^Q&J
+MVP8`1HQT)QPJ63*O;DA6),'UDN(%[CN7']7"?C-^*#[\D:8>[^9$ER5V^\RW
+M4&SXL'?4C.1$UP0&S/H%H*(H1[V">7D"#4T+4&PXM,6J[U"3U3*MZ7,9)45H
+M.8=^6;%?_6,<B@\?=U#B=5FI.ZH&G1<7'Q[7+S`CJR3%VTZY"<6'J^:T;)+%
+MO&D_0O'A$_9:M4=6U34U8U!\>-HF);/R!`X)+7<GW/*Q%2_(ZO&-21>A^/"D
+ME_&!K*?=QRAT'AZV69[;`JEY*\JR4I-"MSH+AVJ.F%)WQ3'/RSKK39&7<K:]
+MXCEST'YXU(O^M:AL0DF?V%?>4`;POJK$27D21PQ9:R\<JFF8PA]J=@`(O&JK
+M+PU["O09TK)55H\%H0O:5+%D0-V$?(.>L=.X"Q('Y(E-Z,#C9T_#:75CVE5U
+MT937=*`$;I@0M7FNLA3'H?/P>;'(6R)_6:_83Z;\N;$P?.$_@7TB]ZS/I[#Q
+M,#1=E1H423U);-$9OQ47AAD_2YWR)*E/H-@P;!&*+,JSYJX'ON]../10X#V1
+K1%;L:[,>V*"*?#3]K6&7R+.947]7@$>VJ)L4?3A?)@````!)14Y$KD)@@@``
+`
+end
--- debian/images/accordion-icon.gif.uu
+++ debian/images/accordion-icon.gif.uu
@@ -0,0 +1,5 @@
+begin 664 accordion-icon.gif
+M1TE&.#EA#``,`/```/___\S,S"'Y!`$```$`+``````,``P```(*C(^IR^T/
+&HYR4%0`[
+`
+end
--- debian/images/panel.png.uu
+++ debian/images/panel.png.uu
@@ -0,0 +1,46 @@
+begin 664 panel.png
+MB5!.1PT*&@H````-24A$4@```)8```%N"`(```#LU79U```'2$E$051XG.W=
+M3VX;1Q!&\5*@A:X47T/P*05>P[F2%P*8Q00$+444.5/=7:_[O540((J17[XB
+MQ]:?I_/Y'$;NK]&_`#N:A/@DQ"<A/@GQ28A/0GP2XI,0GX3X),0G(3X)\4F(
+M3T)\$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ28A/0GP2XI,0
+MGX3X),0G(3X)\4F(3T)\$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A
+M/@GQ28A/0GP2XI,0GX3X),0G(3X)!_3KGU^)'TW"WN7ZA81#^O'WC\2/)F'7
+MTB<8$O9L\\N=8$@X01)VJL4)W9*P1Q>_]"L:$DZ0A,UK=T*W)&S;M5^+*QH2
+M3I"$#>LPP9"P7:U?`B])B$_")GV88+LK&A*VJ-L)W9*P>4TG&!*FUWF"(6%N
+MG_U:3S`DG"`)TQHRP9`PJ_XO@9<D3&B@7TC8KCY7-"*>=_^3I[=3UB_B]>=K
+MUH?JW]@)QI$5)OYW3_R_H7-?^76;8'A()^@0X>)#''Y"MXZN<%G%&WX]KVAX
+M2"<H@7#!(=:98!1<87W%(B^!EW((T0]VB?6?8"2N<)%S6FV"4?"0;M54+.@7
+MN82YY[2FXHV&7-%(7^'$+XHU)QAE#^E6G2%^ZS=J@M&"</%SVK\F*YSLG)8]
+MH5NE#^G6V"'>XS?PBD8[0L]IMP`K'%C]"4930OH0B[\$7FJ[0KKBMPV?8'0Z
+MI/L_QVI8E`E&!\+7GZ_QGO;1^@P1Y!=]5CCK.:UP1:/K.U+(.65-,"*>SN=S
+MGW]3[GH:_0;0_7Y%)A@]5SCK.1U>]T?[PN<4=T*W^AW2K;+G]"&_.E<T^J_0
+M<YK>H-\C+79.N1.,(835'O:A+X&7^']2\;SZ.1U#^-\K8LHY?8\XL,5')UCM
+MBL;`%>:>TWW13^@6_Y!N=3FG!2<88PG'GM,Y)AC#5SCJG$[C%\,)TVMW3FM>
+MT:A`F/YG#M\JSC3!J$`8?3]U>)]?V0E&$<+TEGK8KT+8YYS.-\&H0]BAR5X"
+M+Q4B3![BGP_[L_I%*<+(57S/^6C%KVA4(VS4Q!.,@H190[Q\G"-^]2<8!0E3
+MFNQK5&]7CC#WD6[N$[I5CO!X*2<T(%<TJA$>G^!2)W2K%F%BBTPP)B/,.J&L
+M"A$>O*(+GM"M0H19'9\@Z(I&'<*L"2YU0K>J$!XI]X2R)AA%"+,>YQ><8!0A
+M/%+N"<5-,.B$R[X+O6X\8<H57?.$;HTGW%WZNU#B%8WAA+LGZ`F]!%[AUN(3
+M#"CAR@_RGQM)N.^*>D(_A%KAGU_&ECM![J"'$>Z98-+G%7X55)&TP@]^W#<@
+MN8TA?'B"7WPE<+HB<8B0%38^H=?A%!F$-_P\IP,('[NB=WPSA<7/:?D5=CRA
+MUX$4>Q,^^D;F3K\6YY2B6'B%#WX_FF45"Q,..J&XNA(^=$5W^*TYQ)(K//`M
+MO19\QNA'^,`$BYW0XD.LN,*#?JN=TTZ$]TXPZ7MV+W5.*ZWPN=P)O:[L$"L1
+MIOJM<TY[$([ZCFB+*!9:8=D3^J%JBLT)[YQ@([\5WM<46F&CIC^G)0@I)[1F
+M;0GON:(=_.8>8HD50BNBV)"PR`2W)GY?L]`*9SVGK0A+3;!IPQ6'_MBM[C4Z
+MIV,5%SJD6_.]*#8A_/:*SG%"KQLXQ$$_"'9HDYW3?,+3VZG:3UO^W$SGM,T*
+MO_YY=L,GV+0A0^QZ2$OY37-.DPE9/^]JCG/:;X6E)MBTSD/,)+PQP;)^$YS3
+MY1[M/T<_IVF$_S_!YXC"$VQ:MR&V7.'SK:>+4J'/:4O"]PC.!+F*.81?O9&A
+M^*%KN$*<'W2("82LQ_G;$15;K1`W06Y-"-%^N"$>_6.AF:[H)=;#?OX*T1,D
+M=HCP\P3UZY^_1XIO/^'I[?3R\G+]=YS@D`ZM\/?OWY>_UF]4'E)\.PD_O)%Q
+M@@-+6*%^8]M#..7C/+>C*W2"PSM$J%^%'B;TBE9K_PJ=8)$>([Q,4+\Z^6B/
+M[P%")UBSAU>H7[6>SN?SZ%^#'<K70GP2XI,0GX3X),0G(3X)\4F(3T)\$N*3
+M$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ28A/0GP2XI,0GX3X),0G
+M(3X)\4F(3T)\$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ28A/
+M0GP2XI,0GX3X),0G(3X)\4F(3T)\$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?
+MA/@DQ"<A/@GQ28A/0GP2XI,0GX3X),0G(3X)\4F(3T)\$N*3$)^$^"3$)R$^
+M"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ28A/0GP2XI,0GX3X),0G(3X)\4F(3T)\
+M$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ28A/0GP2XI,0GX3X
+M),0G(3X)\4F(3T)\$N*3$)^$^"3$)R$^"?%)B$]"?!+BDQ"?A/@DQ"<A/@GQ
+?28A/0GP2XI,0W[^F(0R8)L\800````!)14Y$KD)@@@``
+`
+end
--- debian/images/digiumlogo.gif.uu
+++ debian/images/digiumlogo.gif.uu
@@ -0,0 +1,57 @@
+begin 664 digiumlogo.gif
+M1TE&.#EAC``N`/<```````H*"A,3$QL;&R,C(RHJ*C(R,CL[.P`O=``W>0`[
+M>P!`?41$1$I*2E-34UM;6V-C8VMK:W)R<GM[>P!%@@!(A`!.B`!0A0!2BP!8
+MC@E8C0!6D`!;DAQ>D0!AE0!EF`UCE0YGF1!@DAUJER1LFB-RGBAUGC%QGR1T
+MH")YI2MTHC)\H3EVHC6!ISF`IDZ'JD^0L4R8NE.-L5R4LV*4M&.9M6><N&V>
+MO&>AOVVBN7FCOG&CP'FBP'^KP7>VS[0`!;P$$;\`&\$`!,,`#<0`%,4`&\8`
+M(L4"*\4-*<@`),H`*\H,+,().,L`,L03(L82,LH:.M(?.<HF*,8K.\@E.\P<
+M0LDB0,LK0\TK2,TP0\XQ2L\^3\\T7,X\5-$U7-`[7=@Y5=%$7,M+9=1-8--+
+M:]Y!8=%59-%:>,ED<M9A?MQB@MYLAMILB=5TCM5[B]EPA-QWBMU[D>%]F(.#
+M@XN+BY*2DIR<G*.CHZJJJK.SL[N[NX2LP)"SQ9&TRI6ZSIJUR)NZS*W,V;3-
+MV[[-UKS2W*70X->$FMF+GMF0I=F7JN"$G."*G^J"E>.4I>"GMN2LO.JHNNBR
+MOO6BLOFQNN.PP.BTQ,/#P\O+R\K#T=_$SM/3T]O2V=W=W</7X<3;Y</>ZLK;
+MX\+E[<?N]M3DZ]SG[=WI[][L\>C#S^O$T._*U.W3VO+)S_''T_+2U/'1V?+:
+MWO#:XN+BXN7M[NKJZN;O\>WS]N_W^//DZ?3I[?CH[?3P[O/S\_'V^/3Y^_KR
+M]OCU^_CX]_[^_@``````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M````````````````````````````````````````````````````````````
+M`````````````````````"'Y!```````+`````",`"X```C^`(,)'$BPH,&#
+M"!,J7,BPH<.'$"-*-`AJUL2+&#-JW(BPUAY>'$.*'$DR6*\3H$JJ7,ER(8T:
+MMEK*G*D24(>4-'.2!.;JEL^?L#)VTG#CE\ZC(7&)Z1*F:=,N:2Z.(@'"$]*K
+M&QL5*9*D:]<B;232:E%A!]:S&0T5:<*6K1(BE2#R@F$!A56T>"6^(=+6;1%9
+M#WM@N&`VK^&'O,SP[:O$B4-"%CBH&'6X<D-55Y+T;9*D"D-<"CYDL&&Y],)'
+M3Y1L+L)EX8(/'%CL4FCI#ITZ>2Q:UF.G#NZK<I*H[ILD3D(+'F#G0)@'@//G
+MSR'H'LBI`/3KSQEH0FA'`/;G`V;^"]PUY[OS.0;S!("^$,_UA6F,;$Z"Q-%!
+M$QE@GS@(;,#S``($N-YSEA`T"P/.!:#@@@I"AQY!NUS'8(,,B*<)=``*"!TP
+M!%EBW7.7*'1`@@`8P-`8:Q%WQ2L%W8#!!QYPL(=!$2;(`!Z<[!*+'@\,",!V
+M`AWHG`-XV&&DD71$\*$`>0Q4(P`"'#!!'4<:^5LPG#PGP`-ZS*+C'0?X*%XP
+M'@(P(`$)[4(`=`4P],L5\O551!>X#`3(!1_`1L)!")IIQT%Z^&?F0$("4`="
+MLPPHP4`CFEG@0@,6\&A!=#S7P$!E$B!H0A,XUV>;#)VRQ'!M)4%&+L$0LD&>
+ML`%BD";^@O[)W8`A!A-+GW0@!$RG`#Q@D1X#-KF0)0G6>I`$S@W`B4"76#>`
+M'>L]@-"(`;@'`*@,07($J4TH$80DH83`09X<[&D0!,\MM"8`!P2):T)U.-=`
+M+,$XD"Y#'S*PT',3"%1F`+L,>%`>_M%QAW/8,K2MG&5@DL*X>59`")_.2:M0
+M`_?>ZERN_#W@'`0"&>!<!`T]QW%"W@%P*9D?[F(O`+(2%(%SL=B!L$.GI,A6
+M$F!L$@,'R>EYT"XB&[I0><ZYZ]RA!\6[=#":?#AI0K,D./5!+[?+,LU5`X"F
+M@0@.$$R\`22LD!6D)A'%*CX`S:H"@+WZH1X+V9QT,(4:`,'^`WSSS0`#*3LP
+M6Z#.39<0L68NJ]#,[/K[(;WK&COV>G>,?3-#E.BLQ!*3%.(VN28@U*QSDA^$
+MA\!X]VE>LG/HED?*8QZ>H.''.J?UO\L>7")!Z/YH^;4-47%$J8Q\\GF>%B14
+MG7-T*W0'ZH7^)V&_`Q%<.$.(!Z!X0L@VOC4`BCLG@&YE^OJ[V0<A$F<317PA
+M"@<0LXH!'X@6'3-"2`,@D,8`<`R,4<#H'@`JQ*S(,206"6I>0C`V0,?13"`,
+M)%DP*@4`83D-?0:9PG",P`2!`(("K'I;W`S2)Y`IY&7Z2]W&^!.=@11-@OMZ
+MFD($I:_OT4L@SC'1+.PU@%I=<"'^;!B.$HQ`$![@*80?4`!"/':WA*S+8H4Z
+M64&L98!:J:XA`Q*;0A+$L3(!X(;!6$\`[@`L`#SH=R9*B"^FH)FWQ&0@M'A!
+M?D+(@0L`*F65FY5S%!=%A"#03'@0R.EDZ#SQE8X@%!P`^1XW$+LUP&Z!%,@/
+M$[(&U;PE$P;Q!`KB1ZX44(QF'"((,#2Q+BW:ZET'B<6'3L9``%R"=@8!#QB=
+MI(?GP-"+LY3>O`8RR8/`0@M*4$(1#(&00(@@:*SBP`P,,CKG2,`2>K`$-`5H
+M2A7V#U$,/./R/%4'/.`A#WGX)AX>92TST2&:T.21I3K$R('TCI#GHZ01A!F5
+MA/@!F>3^VD`?#&()U7TG``>8Y2P:=<:"[*)WU/-7*\U3`/'4X4/F"8`),;6N
+M[07C$N`1ED`HB,%@I`).12`#0W20`4YR0`2",,@L)N```OBH;`]@FH$>H*G[
+MB;([`RBH0.80`088H`!`#>H`'A!*J$&``0-XZ0$B8-.+-H``!1#H`30E@=C9
+M80`%6)E!X&"$]L5.5S+@)&Q*0!D:6:(WM[G#(04R2FG.LB"SB*8FBDJ06&CB
+M$M*4)EZ!5-<\H-4.>."K0?%JB:_&0IH6Q5LTUXJ**A#A"JIP2"U&(-8,N(`4
+MILGL0.(P!"A$`B*DR``^13.#6FC6-*PX0A$6(1%3A(:.&:!8WVDK`PPW"$$-
+M%QD$`BK`VPI8@`()^,-L#^.*'XPA([Q(KG*32]?A8N47:)!"+YQ+W8BT`@A&
+DJ:YV&>(++R1BN^!5B"O.$-[R'N00YDTO0::KWO:ZU[P!`0`[
+`
+end
--- debian/images/title_gradient.gif.uu
+++ debian/images/title_gradient.gif.uu
@@ -0,0 +1,5 @@
+begin 664 title_gradient.gif
+MB5!.1PT*&@H````-24A$4@````@````;"`8```!!3<Z^````&TE$050HD6/L
+G]F'YSX`',.&3'%4PJF!4`9D*`,YA`A`!X87N`````$E%3D2N0F""
+`
+end
